e2e/tests/metrics-reducer-view.spec.ts

```diff
@@ -91,17 +87,28 @@ test.describe('Metrics reducer view', () => {
 
   test.describe('Sidebar buttons', () => {
     test.beforeEach(async ({ metricsReducerView }) => {
-      await metricsReducerView.gotoVariant('/trail-filters-sidebar');
+      await metricsReducerView.gotoVariant('/trail-filters-sidebar', DEFAULT_STATIC_URL_SEARCH_PARAMS);
+    });
+
+    test.describe('Bookmarks', () => {
+      test('New bookmarks appear in the sidebar', async ({ metricsReducerView, page }) => {
+        const panelTitle = 'deprecated_flags_inuse_total';
+        await metricsReducerView.selectMetricPanel(panelTitle);
+        await metricsReducerView.createBookmark();
+        metricsReducerView.assertBookmarkAlert();
+        await page.goBack();
+        await metricsReducerView.toggleSideBarButton('Bookmarks');
+        metricsReducerView.assertBookmarkCreated(panelTitle);
+      });
     });
 
-    test('Bookmarks', async ({ metricsReducerView }) => {
-      const panelTitle = 'a.utf8.metric 🤘';
-      await metricsReducerView.selectMetricPanel(panelTitle);
-      await metricsReducerView.createBookmark();
-      await metricsReducerView.assertBookmarkAlert();
-      await metricsReducerView.gotoVariant('/trail-filters-sidebar');
-      await metricsReducerView.toggleSideBarButton('Bookmarks');
-      await metricsReducerView.assertBookmarkCreated(panelTitle);
+    test.describe('Group by label', () => {
+      test('A list of metrics is shown when metrics are grouped by label', async ({ page, metricsReducerView }) => {
+        await metricsReducerView.selectGroupByLabel('backend');
```

Comment:
```suggestion
        await metricsReducerView.selectGroupByLabel('code');
```

![image](https://github.com/user-attachments/assets/29049040-8303-440b-85e1-1630bcdc7046)